samples/Cryptocash.Api/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Infrastructure.Persistence/CryptocashDbContext.g.cs

```diff
@@ -116,6 +117,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             if (type != null)
             {
                 ((INoxDatabaseConfigurator)_dbProvider).ConfigureEntity(codeGeneratorState, new EntityBuilderAdapter(modelBuilder.Entity(type)), entity);
+
+                if (entity.Keys.Count == 1 &&
+                    entity.GetAttributesToLocalize().Any())
+                {
+                    type = codeGeneratorState.GetEntityType(entity.LocalizedName);
+                    if (type == null)
+                    {
+                        throw new NullReferenceException($"Type {entity.LocalizedName} is not found in current assembly.");
```

Comment:
May be NotFoundException is more suitable

Reply:
Not needed anymore